server/config/routes.js

```diff
@@ -1064,6 +1066,11 @@ exports.configure = function configure(app) {
   // on the fly tag numbers
   app.get('/lots/generate_barcodes/:number', lots.generateBarcodes);
 
+  app.get('/finance/entities', financeShared.lookupFinancialEntity);
+  app.get('/finance/entities/:uuid', financeShared.lookupFinancialEntityByUuid);
```

Comment:
Are the tests for the /finance/entity and /finance/record API endpoints?

Reply:
Just added some!  This was a good catch too, because I found an unexpected behavior around LIMITs.